Applications/Spire/Include/Spire/Ui/RegionListItem.hpp

```diff
@@ -33,7 +33,7 @@ namespace Spire {
       Nexus::Region m_region;
       Type m_type;
 
-      Type get_type();
+      Type get_type() const;
       TextBox* make_value_label();
```

Comment:
When a conceptual issue is pointed out in a review, look to identify other instances of that issue as well and correct all of them. It's not always feasible in a review to point out every single instance of an issue verbatim.